From 90b175a5eff784472321cb99996fca31d32c5e4b Mon Sep 17 00:00:00 2001
From: Clark Wang <xiaoning.wang@nxp.com>
Date: Thu, 1 Nov 2018 18:57:26 +0800
Subject: [PATCH 4977/5242] MLK-20165-3 dtsi: imx8mm: Add dmas and dma-names
 for ecspi1~3

commit  4d64686567982bd469d5b1489cfc8f7e43c13720 from
https://source.codeaurora.org/external/imx/linux-imx.git

Add the missing attribute dmas and dma-names for ecspi1~3 to fix the
following error log.
LOG: spi_imx 30820000.ecspi: dma setup error -19, use pio

Signed-off-by: Clark Wang <xiaoning.wang@nxp.com>
Reviewed-by: Fugang Duan <fugang.duan@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8mm.dtsi |    6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mm.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8mm.dtsi
index f3d201e..8dd105e 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mm.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8mm.dtsi
@@ -911,6 +911,8 @@
 		clocks = <&clk IMX8MM_CLK_ECSPI1_ROOT>,
 			<&clk IMX8MM_CLK_ECSPI1_ROOT>;
 		clock-names = "ipg", "per";
+		dmas = <&sdma1 0 4 0>, <&sdma1 1 4 0>;
+		dma-names = "rx", "tx";
 		status = "disabled";
 	};
 
@@ -923,6 +925,8 @@
 		clocks = <&clk IMX8MM_CLK_ECSPI2_ROOT>,
 			<&clk IMX8MM_CLK_ECSPI2_ROOT>;
 		clock-names = "ipg", "per";
+		dmas = <&sdma1 2 4 0>, <&sdma1 3 4 0>;
+		dma-names = "rx", "tx";
 		status = "disabled";
 	};
 
@@ -935,6 +939,8 @@
 		clocks = <&clk IMX8MM_CLK_ECSPI3_ROOT>,
 			<&clk IMX8MM_CLK_ECSPI3_ROOT>;
 		clock-names = "ipg", "per";
+		dmas = <&sdma1 4 4 0>, <&sdma1 5 4 0>;
+		dma-names = "rx", "tx";
 		status = "disabled";
 	};
 
-- 
1.7.9.5

